prql-compiler/tests/integration-vendors/docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  postgres:
+    image: "postgres:15-alpine"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: dummy
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+  mysql:
+    image: "mysql:8"
```

Is there any reason to specify version 8?

```suggestion
    image: "mysql:debian"
```